modules/ipi-install-firmware-requirements-for-installing-with-virtual-media.adoc

```diff
@@ -27,7 +27,7 @@ Red Hat does not test every combination of firmware, hardware, or other third-pa
 |====
 | Model | Management | Firmware versions
 | 16th Generation | iDRAC 9 | v7.10.70.00
-| 15th Generation | iDRAC 9 | v6.10.30.00 and v7.10.70.00
+| 15th Generation | iDRAC 9 | v6.10.30.00, v7.10.50.00 and v7.10.70.00
```

Comment:
We should use an Oxford comma here according to the IBM Style Guide. :slightly_smiling_face:

Reply:
Done. Thanks.